frontend/src/views/LoginPage/LoginPage.jsx

```diff
@@ -0,0 +1,69 @@
+import Button from '@mui/material/Button';
+import { TextField, Typography } from "@mui/material"; 
+import { useState } from 'react';
+import './LoginPage.scss';
+import buttonStyles from '../../components/Button/Button.module.scss';
+
+export default function LoginPage () {
+
+    const [username, setUsername] = useState('');
+    const [password, setPassword] = useState('');
+    const [usernameError, setUsernameError] = useState(false);
+    const [passwordError, setPasswordError] = useState(false);
+
+    const handleSubmit = (e) => {
+        e.preventDefault();
+
+        setUsernameError(false);
+        setPasswordError(false);
+
+        if (username && password){
+            console.log(username, password);
+            alert('Login is successful');
+        } else {
+            if (!username){
+                setUsernameError(true);
+            }
+            if (!password){
+                setPasswordError(true);
+            }
+        }
+    }
```

Comment:
Such long functions should be extracted to a separate file or outside the component. 

in the future this function will grow - we will be using backend validation. You may extract it to the src/api/loginValidation.js